src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -63,6 +63,9 @@ public function getProperties($class, array $context = array())
         foreach ($reflectionClass->getMethods(\ReflectionMethod::IS_PUBLIC) as $reflectionMethod) {
             $propertyName = $this->getPropertyName($reflectionMethod->name);
             if ($propertyName) {
+                if (!isset($properties[$propertyName])) {
+                    $propertyName = lcfirst($propertyName);
```

Comment:
Should that be `/^[A-Z]{2,}/` to still lowercase the first letter of methods like `getCarsIOwn()`?